autolens/lens/ray_tracing.py

```diff
@@ -592,7 +592,10 @@ def traced_sparse_grids_of_planes_from_grid(
                 )
                 traced_sparse_grids_of_planes.append(traced_sparse_grids[plane_index])
 
-        return traced_sparse_grids_of_planes
+        if len(sparse_image_plane_grids_of_planes) > 1:
+            return traced_sparse_grids_of_planes, sparse_image_plane_grids_of_planes[1]
+        else:
+            return traced_sparse_grids_of_planes, sparse_image_plane_grids_of_planes[0]

```

Comment:
Try to avoid returning tuples - is it possible that this could be two functions?

Reply:
It is possible... and will be done once I get round to refactoring ray_tracing.py (which is probably next year lol).